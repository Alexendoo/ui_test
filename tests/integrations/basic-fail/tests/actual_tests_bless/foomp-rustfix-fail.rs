//@run-rustfix
#![deny(warnings)]

fn main() {
    let x: String = 42;
    //~^ ERROR: mismatched types
}
