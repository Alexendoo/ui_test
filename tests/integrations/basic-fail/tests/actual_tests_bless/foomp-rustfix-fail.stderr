error[E0308]: mismatched types
 --> $DIR/foomp-rustfix-fail.rs:5:21
  |
5 |     let x: String = 42;
  |            ------   ^^- help: try using a conversion method: `.to_string()`
  |            |        |
  |            |        expected `String`, found integer
  |            expected due to this

error: aborting due to previous error

For more information about this error, try `rustc --explain E0308`.
